npm-shrinkwrap.json

```diff
@@ -4,5929 +4,5487 @@
   "dependencies": {
```

Comment:
@emilgoldsmith Can we add `npm-shrinkwrap.json` to our `.gitignore` and add `npm shrinkwrap` to our build process so we don't have small PRs with 2,000 additions and 2,000 deletions for example?